torch/csrc/Size.cpp

```diff
@@ -1,6 +1,7 @@
 #include "Size.h"
 
 #include <string>
+#include "torch/csrc/utils/python_strings.h"
```

Comment:
Yeah, so `utils.h` is kind of a mess, since it's not separately compilable (you have to include it through THP.h) and pretty much depends on everything. I've been incrementally splitting it up into more fine grained headers. Fine grained so you don't have to include TH or Python headers unless they are necessary and incrementally because I'm too lazy to attempt a wholesale refactor at once.